master/internal/api_generic_tasks.go

```diff
@@ -205,6 +205,7 @@ func (a *apiServer) CreateGenericTask(
 			JobID:      &jobID,
 			LogVersion: model.CurrentTaskLogVersion,
 			Config:     ptrs.Ptr(string(configBytes)),
+			ParentID:   ptrs.Ptr(model.TaskID(*req.ParentId)),
```

Comment:
won't `req.ParentId` be null for root tasks, causing their to panic on a nil pointer dereference?

Reply:
Right! fixed